XamarinCommunityToolkit/Behaviors/MaskedBehavior.shared.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class MaskedBehavior : BaseBehavior
+    {
+        IDictionary<int, char> positions;
+
+        public static readonly BindableProperty MaskProperty =
+            BindableProperty.Create(nameof(Mask), typeof(string), typeof(MaskedBehavior), string.Empty, propertyChanged: OnMaskPropertyChanged);
+
+        public static readonly BindableProperty UnMaskedCharacterProperty =
+            BindableProperty.Create(nameof(UnMaskedCharacter), typeof(char), typeof(MaskedBehavior), 'X', propertyChanged: OnUnMaskedCharacterPropertyChanged);
+
+        public string Mask
+        {
+            get => (string)GetValue(MaskProperty);
+            set => SetValue(MaskProperty, value);
+        }
+        public char UnMaskedCharacter
+        {
+            get => (char)GetValue(UnMaskedCharacterProperty);
+            set => SetValue(UnMaskedCharacterProperty, value);
+        }
+
+        protected override void OnAttachedTo(View bindable)
+        {
+            var inputView = bindable as InputView;
+            inputView.TextChanged += OnEntryTextChanged;
+            base.OnAttachedTo(bindable);
+        }
+        protected override void OnDetachingFrom(View bindable)
+        {
+            var inputView = bindable as InputView;
+            inputView.TextChanged -= OnEntryTextChanged;
+            base.OnDetachingFrom(bindable);
+        }
+
+        static void OnMaskPropertyChanged(BindableObject bindable, object oldValue, object newValue) => ((MaskedBehavior)bindable).SetPositions();
+        static void OnUnMaskedCharacterPropertyChanged(BindableObject bindable, object oldValue, object newValue) => ((MaskedBehavior)bindable).SetPositions();
```

Comment:
Please add an empty line between static function and move **=>** to the next line

```
void Method()
    => ....
```